.travis.yml

```diff
@@ -14,6 +14,8 @@ env:
     - NPROC_MAX=8 BUILDTEST_MCU_GROUP=cortex_m4
 
 before_install:
+    - source ./dist/tools/pr_check/check_labels.sh
+    - check_gh_label "Ready for CI build" || exit 1
```

Comment:
@gebart yes, but don't we need an installed build-environment to perform the static-tests?